src/styles/index.scss

```diff
@@ -1,3 +1,12 @@
+@import 'utils/variables';
+@import 'card';
+@import 'stars';
```

Comment:
Ensure that the `stars.scss` file exists and contains the necessary styles for the stars component.